app/models/master_file.rb

```diff
@@ -640,7 +643,9 @@ def get_ffmpeg_frame_data frame_source, new_width, new_height
         end
         data
       ensure
-        File.unlink(file_source) unless file_source =~ %r(https?://)
+        File.unlink file_source unless file_source =~ %r(https?://)
+        File.unlink frame_source[:source] unless frame_source[:source] =~ %r(https?://)
```

Comment:
Should this be `unless frame_source[:master] or frame_source[:source] =~ %r(https?://)` so it doesn't try to unlink a master file that it didn't create?

Reply:
Good catch, fixed.